pkg/healthcheck/healthcheck.go

```diff
@@ -1342,6 +1342,32 @@ func (hc *HealthChecker) allCategories() []Category {
 						return nil
 					},
 				},
+				{
+					description: "data plane pods labels check",
+					hintAnchor:  "l5d-data-plane-pod-labels",
+					warning:     true,
+					check: func(ctx context.Context) error {
+						pods, err := hc.GetDataPlanePods(ctx)
+						if err != nil {
+							return err
+						}
+
+						return checkMisconfiguredPodsLabels(pods)
+					},
+				},
+				{
+					description: "data plane services labels and annotations check",
```

Comment:
Same goes for this description as well. How about:
`data plane service labels and annotations are configured correctly`